src/components/Initializer.tsx

```diff
@@ -0,0 +1,46 @@
+import dynamic from 'next/dynamic';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { useBosLoaderInitializer } from '@/hooks/useBosLoaderInitializer';
+import { useHashUrlBackwardsCompatibility } from '@/hooks/useHashUrlBackwardsCompatibility';
+import { checkLoggedIn } from '@/redux/slices/account';
```

Comment:
@marcinbodnar  what is the logic behind `@/redux/slices` folder? This folder name is not clear to me

Reply:
Slices are like separate parts of the Redux store, `account` slice is handling `account` part of the Redux store. Here is where the slices (slices reducers) are combined into a single store: https://github.com/near/near-discovery/pull/240/files#diff-b3695b5171edbee1bb22dc8893e3517773bacdd8a51ba2a568e077f6e1e07940R5

In DevTools you see that we have one main object, it's called `account` and that is the part of the store that is handled by `account` slice. 

While expanding this functionality we will be creating more slices.

At this moment in the slice file, we are also storing async thunks and selectors, but if the slice implementation will get bigger it would be a good idea to separate `slice`, selectors, and async thunks into separate files, to make it a bit more readable.